test/graph.js

```diff
@@ -49,38 +49,54 @@ describe('Graph', () => {
           });
       });
     });
-    describe('query: x=0 & y=0', () => {
-      it("should return a 'missing'", (done) => {
+    describe('query: x=230755 & y=6759650', () => {
```

Comment:
On a 1 test "out of bounds" et 2 tests sur des clichés, mais on n'a plus de test 'missing'.
C'est pour le cas d'une couleur qui n'est pas dans le dictionnaire, donc il est normal que cela ne se produire jamais, mais puisqu'on teste le cas dans route/graph.js on pourrait peut-être forcer un pixel d'erreur dans le cache de test?

Reply:
effectué